### PR TITLE
new process design feature: assign documents on process house level

### PR DESCRIPTION
**Pull request by KerstinDrescher**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -201,7 +201,8 @@ nav:
         - Graphical version comparison: admin/features/process-design/graphical_comparison/graphical-comparison.md
         - "Diagram mode: Application Swimlane": admin/features/process-design/swimlane-for-applications/swimlane-for-applications.md
         - "Method: Global responsible role for sub processes": admin/features/process-design/global-responsible-role/global-responsible-role.md
-        - "Method: hide secondary application connections as inputs and outputs": admin/features/process-design/hide-secondary-application-connections/hide-secondary-application-connections.md
+        - "Method: Hide secondary application connections as inputs and outputs": admin/features/process-design/hide-secondary-application-connections/hide-secondary-application-connections.md
+        - "Method: Assign documents on process house and category level": admin/features/process-design/documents-on-processstructure/documents-on-processstructure.md
```
Comment (thomashauser) [inline mkdocs.yml:205]: Method: Assignment of documents on process house and sub category level

File: docs/admin/features/process-design/documents-on-processstructure/documents-on-processstructure.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Method: Documents on process structure level
```
Comment (thomashauser) [inline docs/admin/features/process-design/documents-on-processstructure/documents-on-processstructure.md:1]: Method: Assignment of documents on process house and sub category level

Comment (KerstinDrescher) [reply docs/admin/features/process-design/documents-on-processstructure/documents-on-processstructure.md]: changed

File: mkdocs.yml (hunk 1 above)
Comment (KerstinDrescher) [reply mkdocs.yml]: changed

**Review by Gerrit-at-Symbioworld — APPROVED** [review]

**Pull request merged**